modules/RTC/ScreenObtainer.js

```diff
@@ -138,11 +138,13 @@ const ScreenObtainer = {
                         let audioConstraints = false;
 
                         if (screenShareAudio) {
-                            audioConstraints = {
-                                optional: {
+                            audioConstraints = { };
+
+                            if (this._getAudioConstraints()) {
```

Comment:
Get the result of this and check with typeof if it's boolean and if so, don't destructure it.